secant/Mocked Dependencies/OnboardingMocks.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  OnboardingMocks.swift
+//  secantTests
+//
+//  Created by Francisco Gindre on 9/20/21.
+//
+
+import Foundation
+class OnboardingStepProviderBuilder {
+    var steps: [OnboardingStep] = []
+    var startingAt: Int = 0
```

Comment:
Is this variable used anywhere?

Reply:
thanks, now it is

Reply:
I think you can drop the type here too and just hav startingAt = 0